Marlin/pinsDebug.h

```diff
@@ -57,372 +57,598 @@ static bool report_pin_name(int8_t pin) {
     if (pin == 1) { SERIAL_ECHOPGM("TXD"); return true; }
   #endif
 
-  #if PIN_EXISTS(SERVO0)
-    PIN_SAY(SERVO0_PIN);
+  #if defined(__FD) && __FD > -1   // Pin list updated from 7 OCT RCBugfix branch
+    PIN_SAY(__FD)
```

Comment:
All the `PIN_SAY` lines should end with a semicolon. To get rid of compiler complaints, remove the semicolon I accidentally left at the end of the `PIN_SAY` macro definition.